pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -305,3 +306,84 @@ func TestLookupIPFallback(t *testing.T) {
 	err := f.lookupIP(ctx, "www.google.com")
 	require.NoError(t, err, "Error when resolving name")
 }
+
+func TestString(t *testing.T) {
+	tests := []struct {
+		name           string
+		selectorItem   *fqdnSelectorItem
+		matchName      string
+		matchRegex     string
```

Comment:
These two fields are not used explicitly in the below testcases, so you can remove it.